modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessorTestCase.java

```diff
@@ -227,11 +246,16 @@ public void multipleUpstreamPublishers() throws MuleException, InterruptedExcept
         .subscribe(eventsConsumer2);
 
     eventsEmitter.start();
-    eventsEmitter2.start();
-    eventsConsumer.await();
-    eventsEmitter.stop();
-    eventsConsumer2.await();
-    eventsEmitter2.stop();
+    try {
+      eventsEmitter2.start();
+      eventsConsumer.await();
```

Comment:
in the spirit of reducing flakiness, let's also pass a timeout to `await`

Reply:
Don't we have the overall test timeout for that?
or maybe you are thinking of a shorter timeout for this?

Reply:
shorter for this one

Reply:
using `RECEIVE_TIMEOUT`